ingest/ledgerbackend/captive_core_backend.go

```diff
@@ -139,6 +139,20 @@ func NewCaptive(executablePath, configPath, networkPassphrase string, historyURL
 	}, nil
 }
 
+func (c *CaptiveStellarCore) SetLogger(logger *log.Entry) {
```

Comment:
I think this name better describes what will be logged.
```suggestion
func (c *CaptiveStellarCore) SetStellarCoreLogger(logger *log.Entry) {
```